corehq/apps/app_manager/templates/app_manager/partials/build_errors.html

```diff
@@ -317,5 +317,12 @@
                 {% trans "Errors Building an Application" %}
             </a>
         </p>
+        {% if in_iframe %}
+        <script>
+            $('#build-errors a').each(function(index) {
+                $(this).attr('target', '_parent');
+            });
+        </script>
+        {% endif %}
```

Comment:
didn't love using a script for this, but it would have been a whole bunch of copy paste `if iframe add target`